### DISCUSSION TITLE
Livewire autocomplete not working on mobile using Safari Browser

### DISCUSSION THREAD

**Discussion by nakigami** [livewire/livewire#3243] [1. Help]
Hello everyone, i have a login form in my website, but the problem is that the autofill in mobile using Safari is not working but in Chrome it's working any ideas about this please and thanks in advance.

**Comment by ojsoft** [comment]
I do have a similar issue.

I made a livewire login form and I tried everything thinkable already to convince apple to store the password. Apple evan asks to store the password, but in the end it does not. I also use a livewire registration form and there you think apple stored the suggested password, but next time you want to login, it is not remembered by safari. This drives me nuts.

I tried everything from wire:lazy defere  wire:ignore on the password field, sending the form with a real form post instead of doing a wire:submit.prevent and many more things. I would like to have the livewire functionality on field validation, I integrated a two factor check in login where this comes very handy too, but I can't get Safari Password Manager to save and use the saved passwords on these forms.

I bet the only solution to this is that the livewire team and the apple team work on that problem together.  

Suggestion to lower the pain (Password Manager asking every time if password should be saved, Evan while typing or when another field is changed.) during registration with a fillable password field:

Would be really nice if Livewire would add a wire:onsubmit or something like that for fields that should only be send on submit and completely be left alone on every other request to the network.
